.ddev/config.local.yaml.example

```diff
@@ -7,6 +7,14 @@ hooks:
         > 0 AND mail LIKE '%~%'"
     # This will be applied to production with a future release.
     - exec: drush sqlq "DELETE FROM system WHERE name='hedley_pusher'"
+    - exec: drush updb -y
+    - exec: drush updb -y
```

Comment:
@AronNovak really need this twice?

Reply:
Yep, not sure why, but locally I saw that after one `updb`, there was still one item to process.